src/main/java/duke/Command.java

```diff
@@ -0,0 +1,132 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+/**
+ * Command class will execute the different commands input from the user
+ */
+public class Command {
+    /**
+     * executes different commands from user and save tasks into a file
+     * @param tasks takes in TaskList object containing the list of tasks
+     * @param ui takes in Ui object containing the list of message to user
+     * @param save Storage object to store the TaskList
+     */
+    public void execute(TaskList tasks, Ui ui, Storage save) {
+        ui.Start();
+        boolean hasExit = false;
+        Scanner sc = new Scanner(System.in);
+        String filePath = "data/duke.txt";
+
+        while (!hasExit) {
+            String command = Parser.parseCommand(sc.next());
+            switch(command){
```

Comment:
There should be a white space after the `switch` statement. Generally, Java reserved words should be followed by a white space.

```suggestion
            switch(command) {
```

I noticed the same issue in several other places too.